File: barcodes_file/barcodes_check.R
setwd('/Users/jimblotter/Desktop/Grad_School/Data_Analysis/erisor/barcodes_file/')
install.packages(compare)
library(compare)
before = read.csv("before.csv", header = FALSE)
after = read.csv("after.csv", header = FALSE)
before
after

for(i in before[1,]){
  if(i == after[])      #i is not in after, print "oops"
}

File: QC/barcodes_check.py
before = "/Users/jimblotter/Desktop/Grad_School/Data_Analysis/erisor/QC/before"
after = "/Users/jimblotter/Desktop/Grad_School/Data_Analysis/erisor/QC/after"
import xlrd

workbook = xlrd.open_workbook(r"before.csv")
sheet = workbook.sheet_by_index(0)

col_a = sheet.col_values(0, 1)
col_b = sheet.col_values(1, 1)

my_dict = {a : b for a, b in zip(col_a, col_b)}

print (my_dict)